starboard/loader_app/BUILD.gn

```diff
@@ -23,6 +23,8 @@ _common_loader_app_sources = [
   "system_get_extension_shim.h",
 ]
 
+if ((sb_is_evergreen_compatible && current_toolchain == starboard_toolchain)) {
```

Comment:
this will get built for modular builds: which im guessing you dont want. 

maybe (!sb_is_modular && current_toolchain == starboard_toolchain) is a better condition
same thing applies to a bunch of other conditionals.